test/unit/specs/components/staking/__snapshots__/PageDelegates.spec.js.snap

```diff
@@ -247,7 +247,7 @@ exports[`PageDelegates should filter the delegates 1`] = `
     </div>
   </header>
   <main
-    class="ni-page-main"
+    class="ni-page-main ps"
```

Comment:
`ps`?

Reply:
this is the perfect-scrollbar plugin :/ We need to probably mock it out in tests at some point.